news/PysideLegacyIssues.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* Updated a few deprecated instances of app.exec()
```

Comment:
This section is for things we are deprecating, not deprecations we are fixing.  Please just return this back to the empty news item.